src/main/connector/database/MySQL/MySQL.yaml

```diff
@@ -0,0 +1,359 @@
+extends:
+- ../Database/Database
+connector:
+  displayName: MySQL
+  platforms: Any platform running MySQL
+  information:  Monitors performance and operational metrics for a MySQL database.
```

Comment:
Remove extra white space:
`  information: Monitors performance and operational metrics for a MySQL database.`